packages/protocol/specs/harnesses/GoldTokenHarness.sol

```diff
@@ -11,7 +13,7 @@ contract GoldTokenHarness is GoldToken {
     require(to != address(this), "GoldToken cannot transfer to itself");
     require(value <= balanceOf(msg.sender), "Transfer value exceeded balance of sender");
 
-    address(uint160(to)).transfer(value); // TRANSFER.call.value(0).gas(gasleft())(abi.encode(msg.sender, to, value));
+    address(uint160(to)).sendValue(value); // TRANSFER.call.value(0).gas(gasleft())(abi.encode(msg.sender, to, value));
```

Comment:
ditto on `toPayable`
cc @asaj re implications of changing this

Reply:
https://solidity.readthedocs.io/en/v0.5.3/050-breaking-changes.html

```
Converting address to address payable is possible via conversion through uint160. If c is a contract, address(c) results in address payable only if c has a payable fallback function. 
```

Reply:
we could use zeppelin's function if it makes this line more explicit, no strong opinions here